package.json

```diff
@@ -16,7 +16,7 @@
 		"node": ">=8"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "xo && ava -s"
```

Comment:
Just make the language tests serial, not all the tests. Use `test.serial` instead of `test`.